hopr/hopr-lib/src/lib.rs

```diff
@@ -332,7 +332,11 @@ where
             .store_peer_multiaddresses(&identity.public().to_peer_id(), my_multiaddresses.clone());
     });
 
-    let addr_resolver = DbPeerAddressResolver(db.clone());
+    let addr_resolver = DefaultPeerAddressResolver::new(
+        DbPeerAddressResolver(db.clone()),
+        cfg.transport.peer_map_cache_size,
+        cfg.transport.peer_map_ttl,
```

Comment:
I'd not put this as a `transport`, but rather a dedicated config `db.peer_map.{cache_size,ttl}